src/app/index.html

```diff
@@ -2,7 +2,17 @@
 <html lang="no">
 
 <head>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"> {{{NAV_SCRIPTS}}} {{{NAV_STYLES}}}
+    <title>Foreldrepenger - www.nav.no</title>
+    <meta charSet="utf-8" />
+    <meta name="description" content="Foreldrepenger skal sikre deg inntekt når du ha foreldrepermisjon. Hvis du ikke hatt inntekt, kan du få en engangssum isteden." />
```

Comment:
Description og title skal vel endres. Men så lenge vi husker på det, så kan vi ta det senere.

Reply:
Ja tenkter bedre å ta det senere. Er kommunikasjonsavdelingen som står for innholder der. :)